src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -192,11 +197,11 @@ def execute(self):
                 msg = "WMStatsServer.getProtectedLFNs for T0 is not yet implemented."
                 raise NotImplementedError(msg)
                 # protectedLFNs = set(self.wmstatsSvcT0.getProtectedLFNs())
-                if not protectedLFNsT0:
-                    msg = "Could not fetch the protectedLFNs list from T0 WMStatServer. "
-                    msg += "Skipping the current run."
-                    self.logger.error(msg)
-                    return summary
+                # if not protectedLFNsT0:
```

Comment:
If you want to touch this code, I would then suggest to remove it completely (and the same applies to the service configuration). Otherwise, just leave it as it was before. There is no need to fix all the pylint messages, especially when it's about code that you did not touch within the PR.

Reply:
That's fine with me. The less I change in this PR the cleaner it looks at the end.  I am all for leaving it as it was before.